rdr_common/code_constants.py

```diff
@@ -6,9 +6,9 @@
 # these codes are ignored by RDR.
 PPI_EXTRA_SYSTEM = "http://terminology.pmi-ops.org/CodeSystem/ppi-extra"
 
+LAST_MODIFIED_BUFFER_SECONDS = 60
```

Comment:
Nit: code_constants is for constants used in multiple places in the code. You could probably just define this in participant_summary_dao directly.

Reply:
defined in config.py